src/libraries/Microsoft.PowerFx.Core/IR/IRTranslator.cs

```diff
@@ -93,6 +93,16 @@ public override IntermediateNode Visit(NumLitNode node, IRTranslatorContext cont
                 return MaybeInjectCoercion(node, new NumberLiteralNode(context.GetIRContext(node), value), context);
             }
 
+            public override IntermediateNode Visit(DecLitNode node, IRTranslatorContext context)
+            {
+                Contracts.AssertValue(node);
+                Contracts.AssertValue(context);
+
+                // I think node.NumValue might be dead code, this could be cleaned up
```

Comment:
>// I think node.NumValue might be dead code, this could be cleaned up [](http://example.com/codeflow?start=16&length=69)

copy/paste?

Reply:
Yes, it comes from line 91 above it for NumLitNode.  I figured if it was true there, it is probably true here too.  Not sure if and how it should be cleaned up.